clusterloader2/pkg/prometheus/experimental.go

```diff
@@ -19,17 +19,23 @@ package prometheus
 import (
 	"encoding/json"
 	"fmt"
-	"github.com/spf13/pflag"
-	"k8s.io/apimachinery/pkg/util/wait"
 	"os/exec"
 	"time"
 
+	"github.com/spf13/pflag"
+	"k8s.io/apimachinery/pkg/util/wait"
+
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/klog"
 )
 
+const (
+	kubeProxyManifest = "$GOPATH/src/k8s.io/perf-tests/clusterloader2/pkg/prometheus/manifests/default/experimental/prometheus-serviceMonitorKubeProxy.yaml"
```

Comment:
To make it safe and make sure it doesn't introduce any change when disabled we should gate applying both serviceMonitor and service. Creating such a big service has a performance implications.